dco.md

```diff
@@ -0,0 +1,51 @@
+# Developer Certificate of Origin (DCO)
+
+A [*Developer Certificate of Origin (DCO)*](https://developercertificate.org/)
+is an affirmation that the source code being submitted originated
+from the developer, or that the developer has permission to submit the code. 
+It provides some additional legal protections while being
+relatively easy to do.
+
+The [OpenSSF Charter](https://cdn.platform.linuxfoundation.org/agreements/openssf.pdf>)
```

Comment:
```suggestion
The [OpenSSF Charter](https://cdn.platform.linuxfoundation.org/agreements/openssf.pdf)
```

Reply:
@AevaOnline - oh thanks, you're absolutely right!!